loader_hub/google_drive/base.py

```diff
@@ -122,6 +123,13 @@ def _get_fileids_meta(
             if folder_id:
                 fileids_meta = []
                 query = "'" + folder_id + "' in parents"
+
+                #Add mimeType filter to query
+                if mime_types:
+                    mime_types.append("application/vnd.google-apps.folder") #keep the recursiveness
+                    mime_query = " or ".join([f"mimeType='{mime_type}'" for mime_type in mime_types])
+                    query += f" and ({mime_query})"
+
```

Comment:
should mime_types also be passed to _get_fileids_meta below?

Reply:
You're right. Fixed.